server/db/knexfile.js

```diff
@@ -0,0 +1,52 @@
+require('dotenv').config()
+
+const { CLIENT, DATABASE, PG_USER, PASSWORD } = process.env
+
+module.exports = {
+  development: {
+    client: CLIENT,
+    connection: {
+      database: DATABASE,
+      user: PG_USER,
+      password: PASSWORD
+    },
```

Comment:
Currently we are using the remote database. Can we use this way to get a connection with our database for the production?